mt_dnn/batcher.py

```diff
@@ -34,6 +53,27 @@ def _get_shuffled_index_batches(dataset_len, batch_size):
         random.shuffle(index_batches)
         return index_batches
 
+    @staticmethod
+    def _get_shuffled_index_batches_bin(dataset, batch_size, bin_size, bin_grow_ratio):
+        maxlen = dataset._maxlen
```

Comment:
how about we expose _maxlen as public (rename to be "maxlen") ? I didn't aware _maxlen would be access by other class